corehq/apps/data_dictionary/templates/data_dictionary/base.html

```diff
@@ -47,6 +63,9 @@ <h3 data-bind="text: $root.activeCaseType()"></h3>
           <td>{% trans "Case Property Group" %}</td>
           <td></td>
           <td></td>
+          {% if fhir_integration_enabled %}
+            <td></td>
```

Comment:
This just adds an empty td, in case needed to fill up for empty space in header.